src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,38 @@
+package nextstep.qna.domain;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
+
+public class Answers {
+
+    private final List<Answer> answers;
+
+    public Answers(List<Answer> answers) {
+        this.answers = answers;
+    }
+
+    public void add(Answer answer) {
+        answers.add(answer);
+    }
+
+    public void delete(List<DeleteHistory> deleteHistories, NsUser user) throws CannotDeleteException {
```

Comment:
삭제 이력 객체를 인자로 받기 보다 답변의 삭제 이력을 생성해서 리턴하면 어떨까요?
```suggestion
    public List<DeleteHistory> delete(NsUser user) throws CannotDeleteException {
```